cmd/commands/autofix.go

```diff
@@ -35,20 +47,42 @@ func autofix(cmd *cobra.Command, args []string) {
 	}
 }
 
+func loadKubeAuditConfigFromFile(configFile string) config.KubeauditConfig {
```

Comment:
This looks like it's based on `loadConfigFromFile` from the `all` command. I think that function was meant to use `configFile` (the passed in parameter) instead of `auditAllConfig.configFile` (since the passed in parameter is not used anywhere in the function 🤦 ). I do like `loadKubeAuditConfigFromFile` better as a function name though. So I think what we should do is keep this function but replace `autofixConfig.kubeauditConfigFile` with `configFile`, then remove the `loadConfigFromFile` function, and have the auditAll command call this function instead. Wdyt?

Reply:
totally!! it makes sense! I also failed to notice `configFile` wasn't being used! 😂I will change it and also rename the function!